client/layout/sidebar/custom-icon.jsx

```diff
@@ -13,20 +13,21 @@
  */
 import React from 'react';
 
-const SidebarCustomIcon = ( { alt, className, icon, ...rest } ) => {
```

Comment:
I was worried that removing `alt` would affect accessibility, but I tried out the nav with VoiceOver and it actually sounded better! Currently it says "Jetpack image Jetpack," and with this change it just says Jetpack.